Code/GraphMol/ChemTransforms/MolFragmenter.cpp

```diff
@@ -595,4 +595,301 @@ ROMol *fragmentOnBRICSBonds(const ROMol &mol) {
   return fragmentOnBonds(mol, bondPatterns, &(atomEnvs.get()));
 }
 } // End of MolFragmenter
-} // end of namespace RDKit
+
+namespace {
+const unsigned int NOLABEL = std::numeric_limits<unsigned int>::max();
+// Get the atom label - this might be useful as a util class
+unsigned int get_label(const Atom *a, const MolzipParams &p) {
+  PRECONDITION(a, "bad atom in MolZip::get_label")
+  unsigned int idx = NOLABEL;
+  switch(p.label) {
```

Comment:
the first three cases here all fall through (i.e. there's no `break;` to be found)
Is that intentional?

Reply:
Nope.